core/invoker/src/main/scala/whisk/core/containerpool/docker/HttpUtils.scala

```diff
@@ -133,6 +136,7 @@ protected[core] class HttpUtils(hostname: String, timeout: FiniteDuration, maxRe
   private val connection = HttpClientBuilder.create
     .setDefaultRequestConfig(httpconfig)
     .useSystemProperties()
+    .disableAutomaticRetries()
```

Comment:
we don't use the automatic retries anyway and they don't apply for our operations which use `POST` that is not idempotent.